language-plutus-core/bench/Bench.hs

```diff
@@ -1,13 +1,22 @@
+{-# LANGUAGE OverloadedStrings #-}
+
 module Main (main) where
 
 import           Codec.Serialise
 import           Control.Monad
 import           Criterion.Main
-import qualified Data.ByteString.Lazy       as BSL
-import qualified Data.Text                  as T
+import           Crypto
+import qualified Data.ByteString.Lazy                     as BSL
+import qualified Data.Text                                as T
 import           Language.PlutusCore
+import           Language.PlutusCore.Evaluation.CkMachine (runCk)
```

Comment:
We don't really care about the performance of the CK machine, it's just a reference implementation. We care about the performance of the CEK machine.